torch/_inductor/lowering.py

```diff
@@ -1403,6 +1403,9 @@ def apply_constraint(arg, fx_arg):
 make_fallback(aten.zeros.names)
 
 
+# ROCm specific
+make_fallback(aten.miopen_batch_norm, warn=False)
```

Comment:
We where going to hold off on this correct?

Reply:
This change isn't to register miopen_batch_norm as a decomp as we tried earlier, this instead passes the miopen_batch_norm through the `register_lowering` logic to create the fallback for aten.miopen_batch_norm in the same way unsupported ops have been dealt with in the missing ops tracker https://github.com/pytorch/pytorch/issues/93757
https://github.com/pytorch/pytorch/blob/master/torch/_inductor/lowering.py#L1043

After making this change the generate codes will still to aten.miopen_batch_norm instead of lowering, but will not output this as a warning in the debug log and allows some UTs to pass.

Previously in `test_torchinductor` the batch_norm UTs would fail due to missing lowering
```
torch._dynamo.exc.BackendCompilerFailed: backend='compile_fx_wrapper' raised:
MissingOperatorWithoutDecomp: missing lowering
  target: aten.miopen_batch_norm.default
  args[0]: TensorBox(StorageBox(
    InputBuffer(name='primals_6', layout=FixedLayout('cuda', torch.float32, size=[2, 10, 8, 8], stride=[640, 64, 8, 1]))
  ))
  args[1]: TensorBox(StorageBox(
    InputBuffer(name='primals_1', layout=FixedLayout('cuda', torch.float32, size=[10], stride=[1]))
  ))
  args[2]: TensorBox(StorageBox(
    InputBuffer(name='primals_2', layout=FixedLayout('cuda', torch.float32, size=[10], stride=[1]))
  ))
  args[3]: TensorBox(StorageBox(
    InputBuffer(name='primals_3', layout=FixedLayout('cuda', torch.float32, size=[10], stride=[1]))
  ))
  args[4]: TensorBox(StorageBox(
    InputBuffer(name='primals_4', layout=FixedLayout('cuda', torch.float32, size=[10], stride=[1]))
  ))
  args[5]: False
  args[6]: 0.1
  args[7]: 1e-05
```

At the moment the reason for this passing is not well understood, it could be some additional logic that is added in the make_fallback that is required for lowering in some cases e.g. https://github.com/pytorch/pytorch/blob/master/torch/_inductor/lowering.py#L1040

This needs more investigation and a perf comparison to see if this change will reduce the overhead of creating a fallback during the model run.